.github/workflows/azure-functions-app-powershell.yml

```
@@ -1,49 +1,409 @@
+    environment: ${{ needs.validate.outputs.deployment-environment }}
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+      
+      # Azure login if using RBAC
+      - name: Azure Login (RBAC)
+        if: env.AZURE_RBAC_CREDENTIALS != ''
+        uses: azure/login@v1
```

actionlint (1.7.7)

the runner of "azure/login@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)